src/client/src/containers/AzureFunctionsModal/index.tsx

```diff
@@ -503,7 +605,9 @@ const mapStateToProps = (state: AppState): IStateProps => ({
   appNameAvailability:
     state.selection.services.azureFunctions.appNameAvailability,
   isValidatingName: state.selection.isValidatingName,
-  selection: getFunctionsSelection(state)
+  selection: getFunctionsSelection(state),
+  chooseExistingRadioButtonSelected:
```

Comment:
shouldn't be adding this here, instead: https://github.com/microsoft/WebTemplateStudio/blob/dev/src/client/src/selectors/azureFunctionsServiceSelector.ts then use selection object.

Reply:
I tried this before, but I decided not to use this because `chooseExistingRadioButtonSelected` is a boolean which is not really in the form `label` and `value`, and the dropdown types were really not very flexible. It would require a lot more type changes to get this done, so I could do this in a separate PR if needed. 

Do you happen to know why `selection` is an array by the way? @SahilTara

Reply:
Think it was because at one point it was able to accept multiple functions apps or something

Reply:
Imo the array should be killed off if you dont plan to support multiple function apps being deployed at once.